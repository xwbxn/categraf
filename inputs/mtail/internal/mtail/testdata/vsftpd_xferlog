Mon Feb 21 15:21:32 2011 1 198.51.100.36 528 /incoming/example_324589822_all.deb b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:21:33 2011 1 198.51.100.36 752 /incoming/example_324589822_all.changes b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:26:16 2011 1 198.51.100.36 530 /incoming/example_324589893_all.deb b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:26:17 2011 1 198.51.100.36 752 /incoming/example_324589893_all.changes b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:31:21 2011 1 198.51.100.36 530 /incoming/example_324589969_all.deb b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:31:22 2011 1 198.51.100.36 752 /incoming/example_324589969_all.changes b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:36:11 2011 1 198.51.100.36 528 /incoming/example_324590042_all.deb b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:36:12 2011 1 198.51.100.36 752 /incoming/example_324590042_all.changes b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:41:14 2011 1 198.51.100.36 528 /incoming/example_324590117_all.deb b _ i a anonymous@ ftp 0 * c
Mon Feb 21 15:41:15 2011 1 198.51.100.36 752 /incoming/example_324590117_all.changes b _ i a anonymous@ ftp 0 * c
