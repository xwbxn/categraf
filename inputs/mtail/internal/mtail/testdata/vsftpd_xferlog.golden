counter transfers {direction=incoming} 10 2011-02-21T15:41:15Z
counter transfer_time {direction=incoming} 10 2011-02-21T15:41:15Z
counter bytes_transferred {direction=incoming} 6404 2011-02-21T15:41:15Z
counter connects
counter logins
counter uploads
counter session_time
counter commands {command=}
counter responses {response=}
